cli/command/container/opts.go

```diff
@@ -145,7 +145,7 @@ func addFlags(flags *pflag.FlagSet) *containerOptions {
 		expose:            opts.NewListOpts(nil),
 		extraHosts:        opts.NewListOpts(opts.ValidateExtraHost),
 		groupAdd:          opts.NewListOpts(nil),
-		labels:            opts.NewListOpts(opts.ValidateLabel),
```

Comment:
I think this is the only place   where `ValidateLabel` is called, so I guess we can remove it as well?

Reply:
As it's exported, it might be used elsewhere (for example, I'm pretty sure it's used in`libcompose`). That the only reason why I didn't remove it 😝